lib/rex/post/meterpreter/ui/console/command_dispatcher/core.rb

```diff
@@ -427,6 +430,15 @@ def cmd_channel(*args)
       cmd_read(chan)
     when :write
       cmd_write(chan)
+    when :kill_all
+      if client.channels.empty?
+        print_line('No active channels.')
+        return
+      end
+
+      client.channels.each_pair do |id, channel|
+        cmd_close(id)
```

Comment:
Does this ever raise exceptions? 👀

Reply:
Looks like there's the potential of exceptions being raised in the scenario of a race condition, i.e. the channel is closed just as the `channels -K` call is made